qa/L0_server_status/server_status_test.py

```diff
@@ -191,71 +248,61 @@ def test_model_specific_infer(self):
                            model_version=1, swap=False)
 
             try:
-                for pair in [("localhost:8000", ProtocolType.HTTP), ("localhost:8001", ProtocolType.GRPC)]:
-                    server_status1, req_id1 = _get_server_status(pair[0], pair[1], model_name)
-                    self.assertTrue(model_name in server_status1.model_status,
-                                    "expected status for model " + model_name)
-                    self.assertEqual(len(server_status1.model_status[model_name].version_status), 2,
-                                    "expected status for 2 versions for model " + model_name)
+                for pair in [("localhost:8000", "http"), ("localhost:8001", "grpc")]:
+                    if pair[1] == "http":
+                        triton_client = httpclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+                    else:
+                        triton_client = grpcclient.InferenceServerClient(
+                            url=pair[0], verbose=True)
+
+                    self.assertTrue(triton_client.is_server_live())
+                    self.assertTrue(triton_client.is_server_ready())
+                    model_metadata = triton_client.get_model_metadata(model_name)
+
+                    if pair[1] == "http":
+                        self.assertEqual(model_name, model_metadata['name'])
+                    else:
+                        self.assertEqual(model_name, model_metadata.name)
+
                     for v in (1, 3):
-                        self.assertTrue(v in server_status1.model_status[model_name].version_status,
```

Comment:
instead of loop just test that v1 has stastics and v3 doesn't. Look in lifecycle_test.py for examples of how to do this.